### PR TITLE
simple nofuse build tag 

### PR DESCRIPTION
**Pull request by cryptix**
Although bazillion/fuse#54 should close fairly soon (allowing freebsd fuse support) this gives us a nice chance to make the coupling here as loose as possible to allow moving the code base towards other systems.

### CODE REVIEW

File: core/commands/mount_nofuse.go [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import (
+	"errors"
+
+	cmds "github.com/jbenet/go-ipfs/commands"
+	"github.com/jbenet/go-ipfs/core"
+)
+
+var MountCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Mounts IPFS to the filesystem (read-only) - !!Disabled!!",
```
Comment (jbenet) [inline core/commands/mount_nofuse.go]: maybe just:
```
"Mounts IPFS to the filesystem (disabled)"
```

File: core/commands/mount_nofuse.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+		ShortDescription: `
+Caution! Your version ipfs is compiled _without_ fuse support.
+`,
```
Comment (jbenet) [inline core/commands/mount_nofuse.go]: maybe: 
```
This version of ipfs is compiled without fuse support, which is required 
for mounting. If you'd like to be able to mount, please use a version of 
ipfs compiled with fuse.

For the latest instructions, please check the project's repository: 
  http://github.com/jbenet/go-ipfs 
```

**Comment by jbenet** [comment]
Otherwise, this LGTM!

**Comment by jbenet** [comment]
maybe let's add a `make nofuse`?

File: fuse/ipns/common.go [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	"github.com/jbenet/go-ipfs/core"
+	mdag "github.com/jbenet/go-ipfs/merkledag"
+	nsys "github.com/jbenet/go-ipfs/namesys"
+	ci "github.com/jbenet/go-ipfs/p2p/crypto"
+	ft "github.com/jbenet/go-ipfs/unixfs"
+)
+
+// InitializeKeyspace sets the ipns record for the given key to
+// point to an empty directory.
+func InitializeKeyspace(n *core.IpfsNode, key ci.PrivKey) error {
```
Comment (cryptix) [inline fuse/ipns/common.go]: does this belong in fuse?
cc @whyrusleeping

**Comment by jbenet** [comment]
@cryptix LGTM, Ready To Merge?

**Pull request merged**